activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -841,26 +844,12 @@ def version_string(full_version_string)
           full_version_string.match(/^(?:5\.5\.5-)?(\d+\.\d+\.\d+)/)[1]
         end
 
-        class MysqlString < Type::String # :nodoc:
```

Comment:
@kamipo just a quicks head up. This class was something serialized as part of `AR::Base#to_yaml`, so this PR effectively broke the YAML forward compatibility.

We discussed it with @rafaelfranca, we should either restore that class, or bump the YAML version.

Reply:
Yeah, I'll be going to restore the class, it also broke `Marshal.laod`. Give me an hour or so.

Reply:
No rush. 

Also I don't think Marshal.load compatibility was ever guaranteed, so that's less of a problem.